modules/share/main.lua

```diff
@@ -286,14 +286,14 @@ end
 -- Send player's ultimate and damage via map ping.
 -- This function must always be called from SendAttempt(), which checks whether it's safe to send data.
 local function SendData()
-	local pingType, ultType, ult, dmg, dps = DAMAGE_UNKNOWN, ULT_HORN, 0, 0, 0
+	local pingType, ultType, ult, dmg, dps, lastPingTime = DAMAGE_UNKNOWN, ULT_HORN, 0, 0, 0, GetGameTimeMilliseconds()
```

Comment:
lastPingTime is handled by LibDataShare when entry is nil. But its okay to include it here for consistency